nasa/harmony
# Harmony 1757 - Redact specific headers

## Jira Issue ID
HARMONY-1757

## Description
Gets rid of specific header values like auth, so that they don't show up in our logs.

## Local Test Steps
1. set `TEXT_LOGGER=false` in `.env`
2. Make a request to Harmony like `curl -H "Authorization: Bearer <my-bearer-token>" http://localhost:3000/C123/ogc-api-coverages/1.0.0/collections/bathymetry/coverage/rangeset`
3.  `"authorization":"<redacted>"` should show up in the logs

## PR Acceptance Checklist
